src/components/daily/parametertooltip/ParameterTooltip.js

```diff
@@ -0,0 +1,116 @@
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes, PureComponent } from 'react';
+
+import { formatLocalizedFromUTC, getHourMinuteFormat } from '../../../utils/datetime';
+
+import Tooltip from '../../common/tooltips/Tooltip';
+import colors from '../../../styles/colors.css';
+import styles from './ParameterTooltip.css';
+import i18next from 'i18next';
+
+const t = i18next.t.bind(i18next);
+
+class ParameterTooltip extends PureComponent {
+  renderParameter(parameter) {
+    const prev = (parameter.previousValue !== undefined)
+      ? <div className={styles.previous}>{parameter.previousValue}&rarr;</div>
+      : <div>&nbsp;</div>;
+    return (
+      <div className={styles.pa} key={parameter.id}>
+        <div className={styles.date}>{
+          formatLocalizedFromUTC(
+            parameter.normalTime,
+            this.props.timePrefs,
+            getHourMinuteFormat())
+          }
+        </div>
+        <div className={styles.label}>{t(`params:::${parameter.name}`)} </div>
+        {prev}
+        <div className={styles.value}>{parameter.value}</div>
+        <div className={styles.units}>{t(`${parameter.units}`)}</div>
+      </div>
+    );
+  }
+
+  renderParameters(parameters) {
+    const rows = [];
+    for (let i = 0; i < parameters.length; ++i) {
+      rows.push(this.renderParameter(parameters[i]));
+    }
+
+    return <div className={styles.container}>{rows}</div>;
+  }
+
+  render() {
+    const title = this.props.title ? this.props.title : (
+      <div className={styles.title}>
+        {
+          formatLocalizedFromUTC(
+            this.props.parameter.normalTime,
+            this.props.timePrefs,
+            getHourMinuteFormat())
+          }
+      </div>
+    );
+    return (
+      <Tooltip
+        {...this.props}
+        title={title}
+        content={this.renderParameters(this.props.parameter.params)}
+      />
+    );
+  }
+}
+
+ParameterTooltip.propTypes = {
+  position: PropTypes.shape({
+    top: PropTypes.number.isRequired,
+    left: PropTypes.number.isRequired,
+  }).isRequired,
+  offset: PropTypes.shape({
+    top: PropTypes.number.isRequired,
+    left: PropTypes.number,
+    horizontal: PropTypes.number,
+  }),
+  title: PropTypes.node,
+  tail: PropTypes.bool.isRequired,
+  side: PropTypes.oneOf(['top', 'right', 'bottom', 'left']).isRequired,
+  tailColor: PropTypes.string.isRequired,
+  tailWidth: PropTypes.number.isRequired,
+  tailHeight: PropTypes.number.isRequired,
+  backgroundColor: PropTypes.string,
+  borderColor: PropTypes.string.isRequired,
```

Comment:
```suggestion
  borderColor: PropTypes.string,
```
Not required, there is a default value.